File: PetBuddies-API/PetBuddies-API/Dtos/JanelaAtendimento/JanelaAtendimentoDto.cs
namespace PetBuddies_API.Dtos.JanelaAtendimento
{
    public class JanelaAtendimentoDto
    {
        public int Id { get; init; }

        public DateTime DataHoraInicio { get; init; }

        public DateTime DataHoraFim { get; init; }

        public int DuracaoSlot { get; init; }

        public int VeterinarioId { get; init; }

        public string VeterinarioNome { get; init; } = string.Empty;
    }
}

File: petbuddies-ai/src/main/java/br/com/fiap/petbuddies/dto/client/JanelaAtendimentoDto.java
package br.com.fiap.petbuddies.dto.client;

import java.time.LocalDateTime;

public class JanelaAtendimentoDto {

    private Long id;
    private LocalDateTime dataHoraInicio;
    private LocalDateTime dataHoraFim;
    private Integer duracaoSlot;
    private Integer veterinarioId;
    private String veterinarioNome;

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public LocalDateTime getDataHoraInicio() { return dataHoraInicio; }
    public void setDataHoraInicio(LocalDateTime dataHoraInicio) { this.dataHoraInicio = dataHoraInicio; }
    public LocalDateTime getDataHoraFim() { return dataHoraFim; }
    public void setDataHoraFim(LocalDateTime dataHoraFim) { this.dataHoraFim = dataHoraFim; }
    public Integer getDuracaoSlot() { return duracaoSlot; }
    public void setDuracaoSlot(Integer duracaoSlot) { this.duracaoSlot = duracaoSlot; }
    public Integer getVeterinarioId() { return veterinarioId; }
    public void setVeterinarioId(Integer veterinarioId) { this.veterinarioId = veterinarioId; }
    public String getVeterinarioNome() { return veterinarioNome; }
    public void setVeterinarioNome(String veterinarioNome) { this.veterinarioNome = veterinarioNome; }
}
